mirrord-config/src/agent.rs

```diff
@@ -3,32 +3,52 @@ use serde::Deserialize;
 
 use crate::config::source::MirrordConfigSource;
 
+/// Configuration for the agent pod that is spawned in the kubernetes context.
 #[derive(MirrordConfig, Deserialize, Default, PartialEq, Eq, Clone, Debug)]
 #[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
 #[serde(deny_unknown_fields)]
 #[config(map_to = AgentConfig)]
 pub struct AgentFileConfig {
+    /// Log level for the agent.
+    ///
+    /// Supports anything that would work with `RUST_LOG`.
     #[config(env = "MIRRORD_AGENT_RUST_LOG", default = "info")]
     pub log_level: Option<String>,
 
+    /// Namespace where the agent shall live.
     #[config(env = "MIRRORD_AGENT_NAMESPACE")]
     pub namespace: Option<String>,
 
+    /// Name of the agent's docker image.
+    ///
+    /// You probably don't need this, unless you want to use a custom mirrord-agent build.
```

Comment:
I think we should avoid using pronouns like "You", just keep it formal like, "Not needed, unless a custom mirrord-agent build is required" or something along the lines